concise-mid-tag.cddl

```diff
@@ -138,6 +138,7 @@ $measured-element-type-choice /= uint
 measurement-map = {
   ? comid.mkey => $measured-element-type-choice
   comid.mval => measurement-values-map
+  ? comid.authorized-by => [ + $crypto-key-type-choice ]
```

Comment:
Where is crypto-key-type-choice defined?

Reply:
lines 77-79 in concise-mid-tag.cddl